src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -15,7 +15,7 @@ using namespace winrt::TerminalApp;
 
 // {2bde4a90-d05f-401c-9492-e40884ead1d8}
 // uuidv5 properties: name format is UTF-16LE bytes
-static constexpr GUID c_terminalProfileNamespaceGuid =
+static constexpr GUID TERMINAL_PROFILE_NAMESPACE_GUID =

```

Comment:
Per ISO C++ guidelines, ALL_CAPS is to be avoided except for macros:
[ES.9: Avoid ALL_CAPS names](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es9-avoid-all_caps-names)

The reasoning behind this is that it allows the pre-processor to silently replace you variables with a macro definition without a warning.

Reply:
Chose consistency over correctness after discussion in https://github.com/microsoft/terminal/pull/913#discussion_r286104682 with @miniksa

Reply:
True, but I think we've used the pattern throughout the codebase for constants that used to be #defines. I think the goal here is to favor consistency then write a new issue for ES.9 to be applied codebase-wide with a style change.